app/beacon-omop-worker/scripts/worker.py

```diff
@@ -17,9 +17,15 @@
 filters = subparsers.add_parser("filteringterms", help="Extract filtering terms")
 filters.set_defaults()
 
+individuals = subparsers.add_parser(
+    "individuals", help="Execute queries on individuals"
+)
+individuals.set_defaults()
+individuals.add_argument("--filters", type=str, help="Filtering terms")
```

Comment:
Not saying you have to do it, but you can do something like `individuals.add_argument("-f",  "--filters", type=str, help="Filtering terms")` and have short form args too